step-templates/sitecore-ship-publish.json

```diff
@@ -0,0 +1,114 @@
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "228741aa-9bae-441d-bb2c-1af426863146",
+      "Name": "SiteUrl",
```

Comment:
Same here about adding a prefix all parameter names with `scs` so that we minimize the risk of clashing with other parameters in a users project.